src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -156,6 +152,33 @@ public void SetVersion(BuildScriptGeneratorContext context, string version)
             context.PythonVersion = version;
         }
 
+        public IEnumerable<string> GetDirectoriesToExcludeFromCopyToBuildOutputDir(
+            BuildScriptGeneratorContext context)
+        {
+            var virtualEnvName = GetVirutalEnvironmentName(context);
+            if (!string.IsNullOrEmpty(virtualEnvName))
+            {
+                return new[] { virtualEnvName };
+            }
+
+            return Array.Empty<string>();
```

Comment:
correct me if im wrong some times I saw list instead of array, it will be nice to use list throughout or array throughout ..

Reply:
may not be specific to this PR though